src/language/providers/closeUtils.ts

```diff
@@ -0,0 +1,495 @@
+              break
+            }
+
+            --io
+          }
+        }
+
+        ++ic
+      }
+
+      //if the opening tag is after the trigger and the
+      //closing tag is after the opening tag remove them
+      //both from the array otherwise remove the opening tag
+      io = 0
+      ic = 0
+
+      while (io < openTagArray.length) {
+        if (openTagArray[0] >= triggerPos) {
+          while (ic < closeTagArray.length) {
+            if (closeTagArray[ic] > openTagArray[io]) {
+              closeTagArray.splice(io, 1)
+              openTagArray.splice(io, 1)
+              ic = 0
+              break
+            }
+
+            ++ic
+          }
+        }
+
+        ++io
+      }
+
+      //if the last closing tag is after the trigger position
+      //and the first opening tag is before the trigger position
+      //remove them both
+      ic = closeTagArray.length
+      io = 0
+
+      while (ic >= 0) {
+        if (closeTagArray[ic] >= triggerPos) {
+          while (io < openTagArray.length) {
+            if (openTagArray[io] < triggerPos) {
+              openTagArray.splice(io, 1)
+              closeTagArray.splice(ic, 1)
+              io = 0
+              break
+            }
+
+            ++io
+          }
+        }
+
+        --ic
+      }
+    }
+
+    //if there are any open tags left return the tag item
+    if (openTagArray.length > 0) {
+      return items[i]
+    }
+  }
+
+  return undefined
+}
+
+export function getItemsForLineLT2(
+  document: vscode.TextDocument,
+  triggerText: string,
+  triggerLine: number,
+  nsPrefix: string,
+  items: string[],
+  i: number
+) {
+  let [currentText, currentLine] = [triggerText, triggerLine]
+  let [lineBefore, lineAfter] = [triggerLine, triggerLine]
+  let openTagArray: number[] = []
+  let closeTagArray: number[] = []
+
+  while (
+    currentText.indexOf('<' + nsPrefix + items[i]) === -1 &&
+    currentLine > -1
+  ) {
+    --currentLine
+
+    if (currentLine > -1) {
+      currentText = document.lineAt(currentLine).text
+    }
+
+    if (getItemsOnLineCount(currentText) > 1) {
+      --currentLine
+    }
+  }
+
+  if (currentText.indexOf('<' + nsPrefix + items[i]) > -1) {
+    while (lineBefore > -1) {
+      currentText = document.lineAt(lineBefore).text
+      if (getItemsOnLineCount(currentText) < 2) {
+        if (currentText.indexOf('<' + nsPrefix + items[i]) > -1) {
+          openTagArray.push(lineBefore)
+
+          //if selfclosing remove from the array
+          if (currentText.indexOf('/>') > -1) {
+            openTagArray.splice(openTagArray.length - 1, 1)
+          }
+        }
+
+        if (currentText.indexOf('</' + nsPrefix + items[i]) > -1) {
+          closeTagArray.push(lineBefore)
+        }
+      }
+
+      --lineBefore
+    }
+
+    ++lineAfter
+
+    while (lineAfter < document.lineCount) {
+      currentText = document.lineAt(lineAfter).text
+
+      if (getItemsOnLineCount(currentText) <= 2) {
```

Comment:
The equal sign causes a problem supplying an end tag when the following line contains multiple tags.